packages/s/spin.js.json

```
@@ -13,14 +13,14 @@
     "url": "git://github.com/fgnass/spin.js.git"
   },
   "autoupdate": {
-    "source": "git",
-    "target": "git://github.com/fgnass/spin.js.git",
+    "source": "npm",
+    "target": "spin.js",
     "fileMap": [
       {
         "basePath": "",
         "files": [
           "spin.js",
```

```suggestion
          "*.@(js|css|ts)",
```
There are also some typings included in the package, and as we support those, we might as well bring them in. [Glob](https://www.digitalocean.com/community/tools/glob?comments=true&glob=%2A.%40%28js%7Ccss%7Cts%29&matches=false&tests=%2F%2F%20Imported%20from%20spin.js%20NPM%20package&tests=spin.css&tests=spin.js&tests=package.json&tests=CHANGELOG.md&tests=LICENSE.md&tests=README.md&tests=spin.d.ts&tests=SpinnerOptions.d.ts)

---

Does people include `*.ts` directly on a webpage?

---

Typings aren't used on web pages, but folks that are using Libman or Repl.it for example may want to be able to access typings. We support it, so we might as well include them if they're provided.